cmake/Tools/Platform/Android/android_post_build.py

```diff
@@ -57,23 +57,26 @@ def create_link(src: Path, tgt: Path):
 
     try:
         if src.is_file():
-            tgt.hardlink_to(src)
-            logger.info(f"Created hard link {str(src)} to {str(tgt)}")
+            tgt.symlink_to(src)
```

Comment:
The reason why we used junctions over symlinks on windows was because symlinks seemed to require elevated privileges, at least for folders. This needs to be tested running without elevated privileges